CHANGELOG.md

```diff
@@ -0,0 +1,148 @@
+# Changelog
+
+## [v2.0.4](https://github.com/containeroo/mirrIO/tree/v2.0.4) (2021-04-03)
```

Comment:
since we are adding a new feature, I would suggest bumping the version to v2.1.0

Reply:
true! I've changed it